src/__mocks__/emptyModule.tsx

```diff
@@ -0,0 +1,2 @@
+// Copyright (c) 2018 Ultimaker B.V.
+module.exports = {}
```

Comment:
In some places we use ;, in some places we don't. Let's settle for 1 style at least within this library.